src/coreComponents/constitutive/fluid/multifluid/CO2Brine/CO2BrineFluid.cpp

```diff
@@ -194,11 +194,13 @@ void CO2BrineFluid< PHASE1, PHASE2, FLASH >::checkTablesParameters( real64 const
 template< typename PHASE1, typename PHASE2, typename FLASH >
 void CO2BrineFluid< PHASE1, PHASE2, FLASH >::initializePreSubGroups()
 {
+#if defined(GEOS_DEVICE_COMPILE)
```

Comment:
opposite or I am mssing something?

Reply:
The error is supposed to be issued when it's a GPU run. The model is now activated for CPU only runs.